wfe/wfe_test.go

```diff
@@ -1167,23 +1167,18 @@ func TestChallenge(t *testing.T) {
 
 }
 
-// MockRAStrictUpdateAuthz is a mock RA that enforces authz status in `UpdateAuthorization`
+// MockRAUpdateAuthorizationError is a mock RA that just returns an error on UpdateAuthorization
```

Comment:
This comment says `MockRAUpdateAuthorizationError` but the type is called `MockRAStrictUpdateAuthz`